static/src/javascripts/projects/common/modules/ui/newAutoupdate.js

```diff
@@ -54,7 +54,7 @@ define([
         };
 
         var scrolledPastTopBlock = function () {
-            return $liveblogBody.offset().top < window.scrollY;
+            return $liveblogBody.offset().top < window.pageYOffset;
         };
         var isLivePage = !(window.location.href.search('[?&]page=') !== -1);
```

Comment:
I think you have done it correctly, actually. `window.location.href.search` is a function on the `String` prototype, which is what you want. `window.location.search` is a string of query params.

Question is, how well supported is `window.location.href.search`…?